modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+A `RunStrategy` for virtual machines determines a virtual machine instance (VMI)'s behavior, depending on a series of conditions. The `RunStrategy` setting exists in the virtual machine configuration process as an alternative to the `spec.running` setting.
+The `RunStrategy` setting allows for greater flexibility for how VMIs are created and managed, in contrast to the `spec.running` setting with only `true` or `false` responses.
+
```

Comment:
Add also:
`spec.runStrategy` field is mutually exclusive with `spec.running`

Reply:
Added a sentence "However, the two settings are mutually exclusive and do not affect each other."

Reply:
But they affect. It is not allowed to have both at the same time (will cause an error if the user tries to add both)

Reply:
Ok, I've changed it to "However, the two settings are mutually exclusive. Only either `spec.running` or `spec.runStrategy` can be used because otherwise an error occurs." Is that more accurate?

Reply:
Yes, looks good now